packages/react-microlink/src/components/Card/CardText.js

```diff
@@ -1,17 +0,0 @@
-import React from 'react'
-import styled from 'styled-components'
-import NanoClamp from 'nanoclamp'
-
-const Clamp = ({children, className, lines}) => (
-  <NanoClamp className={className} lines={lines} text={children} is='p' />
-)
-
-const CardText = styled(Clamp)`
-  &&& {
```

Comment:
These style resets are needed.

Also, having `CardText` removed the need for repeating a bunch of props in `CardContent`. I think we should still keep it _extracted_ but maybe in another file location.

Reply:
This is mainly the first reason about why this refactor. Also, it's been tested using gallery microlink demos 🙂